cves/kernel/CVE-2018-10940.yml

```diff
@@ -448,7 +475,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    I believe the mistake here is the unintended check for a wrong arg type. The initial casting of the arg being checked is unsigned int,
```

Comment:
I would try to take a stronger stance here. Using "I believe" feels safe but does not convey confidence in your answer.

Reply:
True, I shouldn't say I believe when I actually know that was the issue. I guess I just wanted to leave it slightly open for others interpretation! But there isn't much to interpret in this vulnerability.